coderd/database/migrations/000279_workspace_app_add_open_in.up.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE workspace_apps ADD COLUMN open_in character varying(64) NOT NULL DEFAULT 'slim-window';
```

Comment:
question about consistency with other fields -

should we define `slim-window` as enum?

Reply:
Yeah I think we should do that for consistency 👍